# More studies

## [UCSC RNA-seq recompute compendium](https://toil.xenahubs.net/) 

TCGA, TARGET, and GTEx RNA-seq data are uniformly re-aligned to hg38 genome, and re-processed using RSEM and Kallisto methods with gencode v23 annotations to generate expression estimates for ~60,000 genes and ~200,000 transcripts, including many LncRNAs. Xena hosts and displays gene and transcript expression results of this analysis.

## [ICGC](https://icgc.xenahubs.net/) 

International Cancer Genome Consortium \(ICGC\) goal is to obtain a comprehensive description of genomic, transcriptomic and epigenomic changes in 50 different tumor types and/or subtypes which are of clinical and societal importance across the globe. It includes TCGA data \(U.S.A.\) plus data contributed by groups from other countries in the International Cancer Genome Consortium. The resource has publically-accessible non-coding somatic mutation data from non-TCGA samples.

## [PCAWG](https://pcawg.xenahubs.net)

The Pan-Cancer Analysis of Whole Genomes \(PCAWG\) study is an international collaboration to identify common patterns of mutation in more than 2,600 cancer whole genomes from the International Cancer Genome Consortium. Building upon previous work which examined cancer coding regions, this project explored the nature and consequences of somatic and germline variations in both coding and non-coding regions, with specific emphasis on cis-regulatory sites, non-coding RNAs, and large-scale structural alterations.

## [GDC](https://gdc.xenahubs.net)

The NCI's Genomic Data Commons \(GDC\) provides the cancer research community with a unified data repository that enables data sharing across cancer genomic studies in support of precision medicine.

The GDC supports several cancer genome programs at the NCI Center for Cancer Genomics \([CCG](http://cancer.gov/aboutnci/organization/ccg/)\), including The Cancer Genome Atlas \([TCGA](https://gdc.cancer.gov/cancer-genome-atlas-tcga)\) and Therapeutically Applicable Research to Generate Effective Treatments \([TARGET](https://gdc.cancer.gov/therapeutically-applicable-research-generate-effective-treatments-target)\), and the Multiple Myeloma Research Foundation study that follow patients over the course of their disease \([MMFR COMMPASS](https://dev.xenabrowser.net/datapages/?cohort=GDC%20MMRF-COMMPASS&removeHub=https%3A%2F%2Fxena.treehouse.gi.ucsc.edu%3A443)\).

## [MET500](https://xenabrowser.net/datapages/?cohort=MET500%20%28expression%20centric%29) 

Xena displays gene expression data from the metastatic cancer study published in [Robinson et al 2017 Integrative clinical genomics of metastatic cancer.](https://www.ncbi.nlm.nih.gov/pubmed/28783718)

## [CCLE](https://xenabrowser.net/datapages/?cohort=Cancer%20Cell%20Line%20Encyclopedia%20%28CCLE%29) 

Cancer Cell Line Encyclopedia. Detailed genetic and pharmacologic characterization of a large panel \(~1100\) of human cancer cell lines.

## Pediatric data

We have a number of sources of pediatric data

### [KidsFirst](https://xenabrowser.net/datapages/?host=https%3A%2F%2Fkidsfirst.xenahubs.net)

The goal of the Gabriella Miller Kids First Pediatric Research Program \(Kids First\) is to develop [a large-scale data resource](https://portal.kidsfirstdrc.org/) to help researchers uncover new insights into the biology of childhood cancer and structural birth defects, including the discovery of shared genetic pathways between these disorders. Over 2015-2018, the program selected 26 patient cohorts for whole genome sequencing through a peer-review process.

### [TARGET](https://gdc.xenahubs.net)

TARGET data is intended exclusively for biomedical research using pediatric data \(i.e., the research objectives cannot be accomplished using data from adults\) that focus on the development of more effective treatments, diagnostic tests, or prognostic markers for childhood cancers. Moreover, TARGET data can be used for research relevant to the biology, causes, treatment and late complications of treatment of pediatric cancers, but is not intended for the sole purposes of methods and/or tool development \(please see [Using TARGET Data](https://ocg.cancer.gov/programs/target/using-target-data) section of the OCG website\). If you are interested in using TARGET data for publication or other research purposes, you must follow the [TARGET Publication Guidelines](https://ocg.cancer.gov/programs/target/target-publication-guidelines).

### [Treehouse Consortium](https://xena.treehouse.gi.ucsc.edu:443)

The goal of the Treehouse Childhood Cancer Initiative \(Treehouse\) is to evaluate the utility of comparative gene expression analysis for difficult-to-treat pediatric cancer patients. Approaching 2000 pediatric tumor data, Treehouse has now assembled a large collection of pediatric cancer RNA-Seq, which, added to adult data, results in a compendium of over 11,000 adult and pediatric tumor-derived gene expression data. Pediatric cancer expression data are from public repository samples and from clinical samples at partner institutions, including UC San Francisco, Stanford, Children’s Hospital of Orange County and British Columbia Cancer Agency. In line with UC Santa Cruz Genomics Institute’s commitment to sharing data and to furthering research everywhere, we have made this data available for all to download and use.

